spark2.4/build-assembly.sbt

```diff
@@ -56,6 +56,7 @@ assemblyExcludedJars in assembly := {
     // Included in / Conflicts with
     // jcl-over-slf4j-1.7.5.jar
     "commons-logging-1.1.1.jar",
+    "commons-lang3-3.7.jar",
```

Comment:
Is this the only change for spark2.4 (compared to spark2.3)?

NB With these duplications I think developing a sbt [AutoPlugin](https://www.scala-sbt.org/1.x/api/sbt/AutoPlugin.html) would make it sharable across builds.

Reply:
In the spark2.4 code, the imports in SpliceDataSource and SpliceRelation were also changed like those in the spark2.3 code.

For AutoPlugin development, wrote [a Jira](https://splicemachine.atlassian.net/browse/DB-10585).